src/components/infinite-scroll/infinite-scroll.ts

```diff
@@ -218,6 +218,15 @@ export class InfiniteScroll {
   }
 
   /**
+  * Pass a promise inside `waitFor()` within the `infinite` output event handler in order to
+  * change state of infiniteScroll to "complete"
```

Comment:
Done. Not sure if I should copy/paste the full example but this way it's probably the most obvious